ui/src/message_popup/renderers/question_interpreter.jsx

```diff
@@ -57,6 +58,29 @@ export default React.createClass({
     }
 
 
+    if (question.notes) {
+      return <MinimalTextResponse
+        key={key}
+        textHeight={192}
+        forceResponse={true}
+        responsePrompt="Notes:"
+        recordText="Next"
+        onLogMessage={onLogMessage}
+        onResponseSubmitted={onResponseSubmitted}
+      />;
+    }
+
+
+    if (question.TimedAutoAdvance) {
```

Comment:
This is great, but the other values (eg., `open` or `notes` or `choices`) are all camelCase (starting with lowercase, no space between and uppercase to start new words).  So please change this to `timedAutoAdvance` to match.